package.json

```diff
@@ -175,7 +176,8 @@
     "testEnvironment": "jsdom",
     "testURL": "http://localhost",
     "moduleNameMapper": {
-      "./src(.*)$": "<rootDir>/src/$1"
+      "./src(.*)$": "<rootDir>/src/$1",
+      "\\/tooltip/styles.(css|scss)$": "identity-obj-proxy"
```

Comment:
Heya @ssvmvss I changed this property so that is specific to the tooltip, I think this way we actually know it's ignoring only the tooltip styles